app/src/main/java/com/example/flashcards/ui/home/HomeFragment.kt

```diff
@@ -95,7 +97,7 @@ class HomeFragment : Fragment() {
     }
 
     private fun addToDirectory(id: Int) {
-        viewModel.send(FlashcardEvent.AddToDirectory(id, 1))
+        homeViewModel.send(FlashcardEvent.AddToDirectory(id, 1))
```

Comment:
To be fixed in order to give a possibility for selecting a directory to add a Flashcard.